extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -892,6 +892,25 @@ export class Bind {
     });
   }
 
+  /**
+   * Skips the subtree at the walker's current node and returns the next node
+   * in document order, if any. Otherwise, returns null.
+   * @param {!TreeWalker} walker
+   * @return {?Node}
+   * @private
+   */
+  skipSubtree_(walker) {
+    let n = walker.currentNode;
+    while (n) {
+      if (n.nextSibling) {
```

Comment:
Please use `walker.parentNode()` and `walker.nextSibling()` only. As is, this will traverse outside of the walker's root and scan unrelated DOM trees.

Reply:
Good catch, done.